deploy_stack.armhf.sh

```diff
@@ -5,5 +5,32 @@ if ! [ -x "$(command -v docker)" ]; then
   exit 1
 fi
 
-echo "Deploying stack"
-docker stack deploy func --compose-file docker-compose.armhf.yml
+if [ $# -eq 0 ]; then
+  command="start"
+else
+  command="$1"
+fi
+
+case "$command" in
+  start)
+    echo "Deploying stack"
+    docker stack deploy func --compose-file docker-compose.armhf.yml
+    ;;
+  stop)
+    echo "Stopping stack"
+    docker stack rm func
```

Comment:
The start/stop/status commands make this look like a service - but the stop here is gonna nuke the stack rather than pause it. Is that what was intended?
Gut feel here is that its confusing the purpose of the script - which from its name is to deploy.
Perhaps generic `generic stop_stack.sh` and `status_stack.sh` scripts instead - they don't need variants I suspect.

Reply:
Agreed, in some ways the deploy script is bringing up a service and the other commands are helping to manage it. Yes, the stop was intended to nuke it...other command name suggestions? And I preferred to put the functionality into a single script given these scripts are at the top level of the repo (repo pollution).